src/components/DotNavigation/DotNavigation.e2e.ts

```diff
@@ -0,0 +1,64 @@
+import { browser } from 'protractor';
+import {
+  createStoryUrl,
+  waitForVisibilityOf,
+} from 'wix-ui-test-utils/protractor';
+import { dotNavigationTestkitFactory } from '../../testkit/protractor';
+import * as eyes from 'eyes.it';
+
+const selectedDotColor = 'rgba(96, 96, 96, 1)';
+
+describe('DotNavigation', () => {
+  const storyUrl = createStoryUrl({
+    kind: 'Components',
+    story: 'DotNavigation',
+    withExamples: true,
+  });
+  const dataHook = 'storybook-DotNavigation';
+
+  beforeEach(() => browser.get(storyUrl));
+
+  eyes.it(
```

Comment:
No need to use `eyes.it` in these tests, just regular `it`.
We prefer to have visual tests in the `*.visual.tsx` file.

Reply:
Sure. The thing is that we cannot take screenshots of these two cases (dot hover and dot click) by just passing props to the component as we do in  <code>*.visual.tsx</code>. If we don't need such screenshots at all, I'll remove them.

Reply:
That's a very good point.
I want to add the ability to be able to do these things inside the `*.visual.tsx` tests (which I hope will be done next week).
But you're right, let's leave it like this for now, and when I'll add the capability, will also merge all the tests from eyes.it to visual.